packages/next-templates/src/app/mijn-omgeving/inloggen/page.tsx

```diff
@@ -0,0 +1,107 @@
+'use client';
+
+import {
+  Article,
+  Button,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Icon,
+  Link,
+  Page,
+  PageContent,
+  Paragraph,
+} from '@utrecht/component-library-react';
+import { UtrechtDigidLogo } from '@utrecht/web-component-library-react';
+import { ExampleHeaderFunnelWmebv } from '@/components/wmebv/Header/ExampleHeaderFunnelWmebv';
```

Comment:
Ik weet het antwoord hier ook niet op en liep hier ook tegenaan: moet hier gebruik gemaakt worden van ExamleHeaderFunnelWmebv of is die alleen voor wmebv? Het alternatief is de ExampleHeader.

Dezelfde vraag voor de het footer component.

Reply:
ExampleHeader is goed, de "funnel header" is een header die minder afleidend is terwijl mensen het formulier invullen.

Reply:
De ExampleHeaderFunnelWmebv heeft een paarse streepje onder de header, de normale header niet. Anders moet ik de navbar implementeren en dat is de hele navbar. 

<img width="500" alt="Screenshot 2024-04-04 at 16 45 43" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861555/53719f75-990b-4436-9668-977e5132c2a8">

Welke hoor ik te gebruiken bij het inlogpagina?